packages/api/emails/decline/html.pug

```diff
@@ -1,29 +1,95 @@
-img(src= `${imgUrl}/email/header.jpg` alt='header' style='width:100%; height:auto;')
```

Comment:
Could you please provide a screenshot of what the email associated with this markup looks like?

Reply:
![Decline Email Screenshot](https://user-images.githubusercontent.com/50112365/122856091-d483a480-d2db-11eb-9e15-f33c98664ba9.png)